examples/xml-attributes/xml_attributes.bal

```diff
@@ -14,26 +14,26 @@ public function main() {
     string s = x1@["{http://ballerina.com/aa}status"];
     io:println(s);
 
-    // An attribute that is not bound to a namespace can be accessed using the string representation of the name.
+    // An attribute that is not bound to a namespace can be accessed using the String representation of the name.
```

Comment:
why are we capitalizing S here?

Reply:
Normally, we capitalize the "S" of Strings.

Reply:
Sorry, normally as in? 

I guess it would make sense in the Java world, but this is just a string of characters here, right?

Reply:
```suggestion
    // An attribute that is not bound to a namespace can be accessed using the string representation of the name.
```